backend/src/packages/chats/chat.service.ts

```diff
@@ -168,6 +168,13 @@ class ChatService implements Service {
     id: number;
     userId: number;
   }): Promise<boolean> {
+    if (!Number(id)) {
```

Comment:
```suggestion
    if (!id)) {
```
id is already number

Reply:
yes, thanks for noticing!